src/openforms/prefill/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class PrefillSkippedException(Exception):
```

Comment:
```suggestion
class PrefillSkipped(Exception):
```
the editor/type checker can already tell it's an exception :) I think a shorter name is nicer.